.gitignore

```diff
@@ -16,3 +16,8 @@ yarn.lock
 dist/
 .eslintcache
 *.tgz
+
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

Comment:
.env*?

Reply:
goot point, but almost, should be `.env*.local`. Fixed.

Reply:
but then this could match `.enviroment.local` or whatever.. maybe it was safer to keep it listed out. Well, hopefully it won't bite us in the future.

Reply:
bikeshed FTW

Reply:
I've reverted it. Being explicit is better.